### DISCUSSION TITLE
Price map

### DISCUSSION THREAD

**Discussion by eliaspfeffer** [electricitymaps/electricitymaps-contrib#5299] [Ideas] [answered]
AFAIK electricity prices are very closed source and only open for big energy traders willing to pay for access. An open source overview would be helpful for policy makers to better understand the electricity distribution (offer and demand).

**Comment by VIKTORVAV99** [comment, marked as answer]
Thanks for opening this discussion and providing your thoughts!

While a price map would indeed be a nice addition we have run into regulatory and licensing issues with the redistribution of some of the price data and have therefore been forced to remove it for now. And without this data there simply no way for us to create such a map or provide said price data.

We are however working on finding alternative sources for the price data that would allow us to redistribute the data, once that is done we can look into adding either a price map or some other form of visualization to convey the price of electricity in a region.

**Reply by eliaspfeffer** [reply]
Thank you for your reply. What a pitty, that this is so closed source. But I’ve found one website / GitHub Projekt, where they have at least published the European map: https://thingler.io/map
What would be helpful is an American map as well. Especially one with historic data

**Reply by VIKTORVAV99** [reply]
That map use the same source for its data as we do. I don't want to get into what affects or may affect other projects but the ENTSO-E transparency platform does not list the price data as free to re-use.
